{
  willThrow(RuntimeException.class).given(mock).voidMethod();
  try {
    mock.voidMethod();
    fail();
  }
 catch (  RuntimeException e) {
  }
}
